src/System.Buffers.Experimental/System/Buffers/IReadOnlyBufferList.cs

```diff
@@ -11,5 +11,7 @@ public interface IReadOnlyBufferList<T> : ISequence<ReadOnlyMemory<T>>
         ReadOnlyMemory<T> First { get; }
 
         IReadOnlyBufferList<T> Rest { get; }
+
+        long Length { get; }
```

Comment:
Why is the length int64?

Reply:
The property is the total length of Ts in the list. It can be larger than int.MaxValue.